doc/requirements/requirements.md

```diff
@@ -0,0 +1,306 @@
+ME007 | Mission Execution.Logging | 1 | The Mission Execution module SHOULD log activity for later analysis. | In case of forensic analysis of a safety event, for example.
+ME008 | Mission Execution.Feedback.Inputs.Error Notification.Automatic Error Recovery | 1 | Upon receipt of a downstream failure (from a Global Planner), the Mission Execution module SHOULD attempt to automatically recover from the error | For example, if the robot gets stuck, the Mission Execution module could have the robot perform a recovery maneuver of some kind, before continuing with execution of the mission plan.
+ME009 | Mission Execution.Feedback.Outputs.Error Information | 1 | If the Mission Execution module is unable to execute the mission, it MUST report the failure | This would be received by the higher-level software. Perhaps a remote operating center where the remote operator "rescues" the robot. 
+ME010 | Mission Execution.Safe State Upon Failure | 1 | If the Mission Execution module is unable to execute the mission, it MUST direct the robot to a safe state. | *TODO: Define "safe state"* The failure could be for a variety of reasons - sensor failures, algorithmic failure, a collision, etc.
+
+### 2.4.3 Global Planning
+
+The Global Planning module compute the intended route for the robot, typically using a static map, the robot's initial pose, and the desired primitive to execute. 
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+GP001 | Global Planning | 1 | The Navigation System SHOULD have a Global Planning module which generates the route for the robot to follow for the specified primitive, respecting any policy guidance contained in the Mission Plan | Could use pluggable cost functions 
+GP002 | Global Planning.Inputs.Static Map | 1 | The Global Planner SHALL have a static map available that describes the environment | Assumes the map has already been created and available as an input
+GP003 | Global Planning.Inputs.Current Pose | 1 | The Global Planner SHALL have the robot's current pose | The pose could be be provided manually or automatically determined (outside of this module)
+GP004 | Global Planning.Inputs.Primitive to Execute | 1 | The Global Planner SHALL have the primitive operation to execute
+GP005 | Global Planning.Inputs.Policy | 1 | The Global Planner SHALL have policy guidance associated with the primitive operation to execute | This could be global policy and/or per-primitive policy
+GP006 | Global Planning.Multiple Global Planners | 1 | As part of the Navigation System, there SHOULD be multiple global planners available and chosen depending on the primite to be executed | Dynamic selection of global planners
```

Comment:
In supporting this, I think you'll find that you also want to make it easy to "reset or disable" a planner. Most of these planners may have large memory usage, and for efficiency may not want to recreate them for each plan -- if you have multiple running at once, each taking up memory, you might actually run out of memory quickly. Having an easy API to say "hey, not using this planner for a while, have it release resources" would be very useful.

Reply:
Do you think that's worth the complexity versus simply loading and unloading them? If so, we can introduce this intermediate "disabled" state with the appropriate calls/messages.